docs/spec/bank/keeper.md

```diff
@@ -0,0 +1,22 @@
+# Keeper
+
```

Comment:
I also think it would help to add an example here for a token with the default metadata.